dvc/tree/base.py

```diff
@@ -77,6 +77,19 @@ def __init__(self, repo, config):
         self.verify = config.get("verify", self.DEFAULT_VERIFY)
         self.path_info = None
 
+    @cached_property
+    def jobs_count(self):
+        """Number of workers per cpu core."""
+
+        # NOTE: if there wasn't the same snippet here I'd rather do it in the following
+        # style:
+        # repo_config = self.repo.config['core'] if self.repo else {}
+        # jobs_per_core = self.config.get('jobs', repo_config.get('jobs', self.JOBS))
+        jobs_per_core = self.config.get("jobs") or (
+            (self.repo and self.repo.config["core"].get("jobs")) or self.JOBS
+        )
```

Comment:
Existing --jobs and cls.JOBS were never about jobs-per-core, but rather an absolute number of jobs. This breaks SSH, for example, which has JOBS=4 but now it will spawn 4*NCPU

Reply:
Thank you! It is clear now.